tracing-api/src/main/java/com/palantir/tracing/api/OpenSpan.java

```diff
@@ -83,26 +87,25 @@ public static Builder builder() {
         return new Builder().startTimeMicroSeconds(getNowInMicroSeconds()).startClockNanoSeconds(System.nanoTime());
     }
 
-    /** Use this factory method to avoid allocate {@link Builder} in hot path. */
+    /**
+     * Use this factory method to avoid {@link Builder} allocations in hot paths.
+     *
+     * @deprecated Use the variant without an originating span id
+     */
+    @SuppressWarnings("InlineMeSuggester")
```

Comment:
What's this magic?

Reply:
Error prone has an inlineme annotation to document replacements on undesirable call sites. We don’t want to automate this migration because there’s potentially other data that can be removed.